code/renderers/vue3/template/stories/decorators.stories.js

```diff
@@ -55,12 +60,19 @@ export const DynamicWrapper = {
     // Number type is detected, but we still want to constrain the range from 1-6
     level: { control: { type: 'range', min: 1, max: 6 } },
   },
+  decorators: [DynamicWrapperWrapper],
+};
+
+export const MultipleWrappers = {
+  args: { label: 'With multiple wrappers' },
+  argTypes: {
+    // Number type is detected, but we still want to constrain the range from 1-6
+    level: { control: { type: 'range', min: 1, max: 6 } },
+  },
   decorators: [
-    (storyFn, { args }) => ({
-      template: `<div :style="{ borderWidth: level, borderColor: 'red', borderStyle: 'solid' }"><story /></div>`,
-      data() {
-        return { level: `${args.level}px` };
-      },
-    }),
+    ComponentTemplateWrapper,
+    SimpleTemplateWrapper,
+    VueWrapperWrapper,
+    DynamicWrapperWrapper,
   ],
 };
```

Comment:
@kasperpeulen this stories cover almost all  cases, i have done some changes and add story that stacks all of them which works perfect